samples/net/coap_server/src/coap-server.c

```diff
@@ -131,6 +131,41 @@ static int well_known_core_get(struct coap_resource *resource,
 	return r;
 }
 
+static void msg_dump(const char *s, u8_t *data, unsigned len)
+{
+	unsigned i;
+
+	printk(" msg_dump =  %s \n ", s);
+	for (i = 0; i < len; i++)
+		printk("%02x ", data[i]);
+
+	printk("\n");
+
+	for (i = 0; i < len; i++)
+		printk(" %c ", data[i]);
+
+	printk("\n %u bytes \n \n", len);
+}
+
+static void paload_dump(const char *s, u8_t *data, unsigned len,unsigned payloadlen)
+{
+	unsigned i,LoopCount;
```

Comment:
I believe Zephyr is not using CamelCase naming convention.
Alter "LoopCount" to loopcount or loop_count